src/rcx/src/extRCmodel.cpp

```diff
@@ -760,7 +760,7 @@ extDistWidthRCTable::extDistWidthRCTable(bool over,
   _firstDiagDist = nullptr;
   _lastDiagDist = nullptr;
 
-  for (jj = 0; jj < 16; jj++) {
+  for (jj = 0; jj < 32; jj++) {
```

Comment:
Why is 32 more correct than 16?  What do either represent?

This value is to be used in multiple places so it should be a named constexpr static class member.